### ISSUE TITLE
Sent links are not visible

### ISSUE THREAD

**Issue by felixbade** [felixbade/chatgpt-web-ui#39]
Not WCAG compatible :D 
![435E87BA-6EBC-4271-A24B-00C0BAC29BFD](https://user-images.githubusercontent.com/6554998/222898538-23f2829e-f2b8-435e-a245-b9cadab31780.jpeg)
![9B8F0EDC-EF0A-473E-8E3C-4988671E7F21](https://user-images.githubusercontent.com/6554998/222898540-102d21ae-6bb3-4eff-9071-ab8dc653e6b5.jpeg)

Looking at Signal and Telegram, they have solved this issue by making sent links the same color as normal text and adding an underline. iMessage doesn’t display links at all, only the rich preview.

Signal and Telegram also have an underline in the received links. Telegram uses an accent color for the text where as Signal doesn’t.

**Comment by felixbade** [comment]
Fixed in 957cc8b
